plugins/prompt.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: prompt
+spec:
+  platforms:
+  - head: https://github.com/jordanwilson230/kubectl-plugins/archive/krew.zip
+    uri: https://github.com/jordanwilson230/kubectl-plugins/archive/v1.0.0-krew.zip
+    sha256: efa1c2122ecaeb0a43ba1ed9e54c1ff1274d20de76934f7bb91dd154f2327aa1
+    bin: kubectl-prompt
+    files:
+    - from: "*/kubectl-prompt"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  version: "v1.0.0-krew"
+  caveats: When removing this plugin, delete the line beginning with `function kubectl()` in your ~/.bash_profile
+  shortDescription: Prompts for user confirmation when executing commands in critical namespaces or clusters, i.e., production.
+  description: Protect your production environments! When issuing certain commands inside a specified namespace/cluster, a warning prompt appears, telling the user they are in a flagged environment and asks to confirm the command. Ex., `kubectl prompt`, `kubectl prompt add -n production`, kubectl prompt add -c `prod-cluster`. `kubectl prompt list`.
```

Comment:
can you wrap these lines by using yaml multiline string https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines

Reply:
Done